data/human/human missions.txt

```diff
@@ -7482,9 +7482,9 @@ mission "Lost Dog"
 			`	The butler's frown deepens as you show Caesar to him.`
 			`	He sighs and says, "Card, please." He holds out a silver tray and awaits your calling card.`
 			choice
-				`	(Explain you don't have a calling card.)`
+				`	(Explain I don't have a calling card.)`
```

Comment:
"Explain I don't have a calling card" just seems kind of awkward to me in a way the original doesn't. Is there some way to rephrase this?